.clang-format

```diff
@@ -1,16 +1,35 @@
----
 Language: Cpp
-BasedOnStyle: LLVM
-AccessModifierOffset: -8
-BreakBeforeBraces: Allman
+BasedOnStyle: Google
+AccessModifierOffset: -4
+BreakBeforeBraces: Attach
```

Comment:
this would mean:

```c
struct finfo {
	file_info_t info;
	char16_t filename[256];
};
```

Instead of:

```c
struct finfo 
{
	file_info_t info;
	char16_t filename[256];
};
```

IMO this should improve our code readability. Moreover, it would also avoid noisy frictions when we switch from other languages like Go/Rust. But this is just my opinion, let me know :)